p2p/src/authenticated/lookup/actors/tracker/ingress.rs

```diff
@@ -194,6 +194,7 @@ impl<E: Spawner + Metrics, C: PublicKey> Oracle<E, C> {
     ///   Should be monotonically increasing.
     /// * `peers` - Vector of authorized peers at an `index` (does not need to be sorted).
     ///   Each element is a tuple containing the public key and the socket address of the peer.
+    ///   The peer must be dialable at and dial from the given socket address.
```

Comment:
We could allow the caller to provide both dialable/dialer IP (if different) but that seemed like overkill at this stage.